src/dftbp/dftbplus/input/geoopt.F90

```diff
@@ -184,6 +189,24 @@ subroutine readOptTolerance(node, input)
   end subroutine readOptTolerance
 
 
+  !> Entry point for reading input for SteepestDescent
+  subroutine readSteepDescInput(node, input)
+
+    !> Node to get the information from
+    type(fnode), pointer, intent(in) :: node
+
+    !> Control structure to be filled
+    type(TSteepDescInput), intent(out) :: input
+
+    type(fnode), pointer :: field
+    type(string) :: modifier
+
+    call getChildValue(node, "StepSize", input%stepSize, 100.0_dp, modifier=modifier, child=field)
+    call convertUnitHsd(char(modifier), timeUnits, field, input%stepSize)
```

Comment:
The only reason I kept it is the old driver. We should discuss if we want to keep this StepSize or switch to something less cumbersome, especially since calculating the weights requires the masses and messes up the function signature(s).

Reply:
One for the next dev meeting. My quick test suggested the unweighted coordinates required fewer steps for H2O2, but the weighted ones were better for H2O.